### PR TITLE
Add setup steps for running MAUI sample app on VS Code

### PR DESCRIPTION
**Pull request by DharshanBJ**
Add setup steps for running MAUI sample app on VS Code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,27 @@ To run this sample, you'll need:
 - A user account in your Microsoft Entra tenant. This sample will not work with a Microsoft account (formerly Windows Live account). Therefore, if you signed in to the [Microsoft Entra admin center](https://entra.microsoft.com) with a Microsoft account and have never created a user account in your directory before, you need to do that now.
 - Setup [MAUI](https://learn.microsoft.com/dotnet/maui/get-started/installation?tabs=vswin&view=net-maui-7.0). This will require Visual Studio on PC, and on a Mac Machine.
 
+To run the sample on VS Code, you'll need:
+
+- For iOS/Mac, A specific version of Xcode, which depends on the version of .NET MAUI that you're using is needed. For information, see [Release Versions](https://github.com/dotnet/maui/wiki/Release-Versions), make sure to install the iOS simulations during the xcode installation.
```
Comment (ashok672) [inline README.md:52]: Nit. Simulators instead of simulations

File: README.md [hunk 2]
```diff
@@ -47,6 +47,27 @@ To run this sample, you'll need:
...
+- [VS Code](https://code.visualstudio.com/download). Install or update Visual Studio Code.
+  - In Visual Studio Code, in the Extensions tab, search for ".NET MAUI" and install the .NET MAUI extension. The .NET MAUI extension automatically installs the C# Dev Kit and C# extensions, which are required for the .NET MAUI extension to run.
+- Install [.NET](https://learn.microsoft.com/en-us/dotnet/core/install/windows)
+- Install the .NET MAUI workload.
+  - Windows - dotnet workload install maui
+  - Mac - sudo dotnet workload install maui
+- While trying to build and launch the MAUI project on Mac, if you run into "Please Verify the Xcode Installation" Error, do the following:
+  - In your Terminal, first run:
+    $ /Applications/Xcode.app/Contents/Developer/usr/bin/simctl delete unavailable
+  - Reboot, then reopen your MAUI solution
```
Comment (ashok672) [inline README.md:62]: Reboot VS code right not machine reboot? If so, just call it out explicitly

File: README.md [hunk 3]
```diff
@@ -47,6 +47,27 @@ To run this sample, you'll need:
...
+- For iOS/Mac, setup provisioning on your device by following the below steps:
+  - Open Xcode -> Settings -> Accounts, Add your Microsoft Apple ID
+  - Click on "Download Manual Profiles" and wait for it to finish
```
Comment (ashok672) [inline README.md:65]: Also add dowloading certificates in addition of downloading manual profiles.

File: README.md [hunk 4]
```diff
@@ -47,6 +47,27 @@ To run this sample, you'll need:
...
+- Open the MAUI project in VS Code, create .vscode/launch.json from the root folder and add ".NET MAUI" as the name of the launch configuration.
+- For building the MAUI project, right click on the project in Solution Explorere and select "Build".
```
Comment (ashok672) [inline README.md:67]: Spelling. Explorer

**Review by ashok672 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (DharshanBJ) [reply README.md:52]: updated

File: README.md (hunk 2 above)
Comment (DharshanBJ) [reply README.md:62]: its machine reboot, stated this explicitly

File: README.md (hunk 4 above)
Comment (DharshanBJ) [reply README.md:67]: corrected

File: README.md (hunk 3 above)
Comment (DharshanBJ) [reply README.md:65]: updated

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**